src/Middleware/LoginSessionController.php

```diff
@@ -2,13 +2,10 @@
 
 namespace SilverStripe\SessionManager\Control;
```

Comment:
The namespace for this file doesn't match its location. Putting this under "middleware" makes no sense

Reply:
Doesn't look like this file is covered by any tests.

Reply:
I've already raised a [separate issue](https://github.com/silverstripe/silverstripe-session-manager/issues/42) about the somewhat messed up folder/namespace situation, trying to limit the scope of this PR

Reply:
I've create a [new issue ](https://github.com/silverstripe/silverstripe-session-manager/issues/45) to write unit tests for LoginSessionController, and do the status codes at the same time